terraform/data.tf

```diff
@@ -50,6 +50,7 @@ data "template_file" "container_definition" {
     project           = local.project
     environment       = var.environment
     aws_region        = var.region
+    name_suffix       = local.name_suffix
```

Comment:
A lot of times we need to add a character limit to the name suffix since feature branch names can be quite long. Do you think we need to worry about that for database names?

Reply:
Good question - postgres has 63 character limit for db names which is probably safe.